scala-api/src/test/scala/com/pagerduty/scheduler/SchedulerClientSpec.scala

```diff
@@ -65,7 +76,7 @@ class SchedulerClientSpec extends WordSpecLike with Matchers with ScalaFutures w
       val partitionId = task.partitionId(kafkaNumPartitions)
 
       "successful" should {
-        val result = new RecordMetadata(new TopicPartition(kafkaTopic, 0), 0L, 0L)
+        val result = new RecordMetadata(new TopicPartition(kafkaTopic, 0), 0L, 0L, 0, java.lang.Long.valueOf(0L), 0, 0)
```

Comment:
Same question as above

Reply:
The new version of the library seemed to have breaking changes